app/assets/javascripts/discourse/app/templates/components/discourse-banner.hbs

```diff
@@ -1,12 +1,18 @@
 {{#if visible}}
   <div class="row">
     <div id="banner" class={{overlay}}>
-      {{d-button icon="times" action=(action "dismiss") class="btn btn-flat close" title="banner.close"}}
-      <div id="banner-content">
-        {{html-safe content}}
+      <div class="floated-buttons">
         {{#if currentUser.staff}}
-          <p><a href={{banner.url}}>{{html-safe (i18n "banner.edit")}}</a></p>
+          <a href={{banner.url}} class="btn btn-flat edit-banner">
+            {{d-icon "pencil-alt"}}
+            {{html-safe (i18n "banner.edit")}}
```

Comment:
I think using only the pencil icon with no label would be fine here

Reply:
On desktop, I find it's nice to have the label. But makes sense on mobile, removed the text label there: 

<img width="327" alt="image" src="https://user-images.githubusercontent.com/368961/158364093-bbb861c9-76b4-49c6-bbbc-4ca0f462e219.png">